Api/WannaSpeakHttpClient.php

```diff
@@ -83,6 +91,10 @@ public function createAndSendRequest($args, $headers = [], $body = null)
         $uri     = $uri->withQuery(http_build_query($args));
         $request = MessageFactoryDiscovery::find()->createRequest(self::DEFAULT_METHOD_POST, $uri, $headers, $body);
 
+        if (null !== $this->logger) {
+            $this->logger->info('Sent request : '.$request);
```

Comment:
Tes logs ne sont pas contextualisés, ils vont se perdre dans le flot des logs de l'app
```suggestion
            $this->logger->info('WannaSpeak Request sent : '.$request);
```